java/src/main/java/com/google/cloud/dataproc/templates/hbase/README.md

```diff
@@ -0,0 +1,94 @@
+## 1. HBase To BigQuery
+### Required JAR files
+
+Some HBase dependencies are required to be passed when submitting the job. These dependencies are automatically set by script when CATALOG environment variable is set for hbase table configuration. If not, 
+these dependencies need to be passed by using the --jars flag, or, in the case of Dataproc Templates, using the JARS environment variable. 
+- **[Apache HBase Spark Connector](https://mvnrepository.com/artifact/org.apache.hbase.connectors.spark/hbase-spark) dependencies (already mounted in Dataproc Serverless, so you refer to them using file://):**
+    - file:///usr/lib/spark/external/hbase-spark-protocol-shaded.jar
+    - file:///usr/lib/spark/external/hbase-spark.jar
+    - All other dependencies are automatically downloaded and set once CATALOG environment variable is used for hbase table configuration. Lib link - [hbase-client](https://repo1.maven.org/maven2/org/apache/hbase/hbase-client/2.4.12/hbase-client-2.4.12.jar), [hbase-shaded-mapreduce](https://repo1.maven.org/maven2/org/apache/hbase/hbase-shaded-mapreduce/2.4.12/hbase-shaded-mapreduce-2.4.12.jar)
+    
+  ### Configure the [hbase-site.xml](./hbase-site.xml)
+    
+    
+1) **Automatic process of creating custom container**-: The process is automatically done in the start-up script, when environment variable HBASE_SITE_PATH is set.
+2) **Configure the [hbase-site.xml](./hbase-site.xml) manually and create container**
```

Comment:
Do user still need to edit hbase-site.xml ?

Reply:
Yes, but the hbase-site.xml here is for reference. Users may have there own files as well in a local path that can be used to create container image. Hence the two options.